apps/ui/src/app/(error)/error.tsx

```diff
@@ -1,18 +1,30 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { View, Text } from 'react-native';
 import tw from 'twrnc';
-import { useAuthStore } from '../../state/useStore.ts';
+import { useAuthStore } from '../../state/useStore';
+import { router } from 'expo-router';
 
 const ErrorComponent = () => {
   const error = useAuthStore(
     (s) => s.registerError || s.loginError || s.generalError,
   );
 
+  useEffect(() => {
+    const timeout = setTimeout(() => {
+      useAuthStore.setState({ generalError: null });
+      router.push('/sign-up');
+    }, 5000);
+
+    return () => {
+      clearTimeout(timeout);
+    };
+  }, [error]);
+
   return (
-    <View style={tw` h-screen bg-black`}>
+    <View style={tw` bg-black`}>
```

Comment:
Extra space here

Reply:
lol.. :D it is not affecting the app in some way but it is indeed an extra space there :P